src/components/nav/desktop/NavColumn.js

```diff
@@ -182,11 +182,15 @@ const NavColumn = ({
     if (route) goToRoute(route)
   }, [goToRoute, getTrackPageLink])
 
-  const onShowVisualizer = useCallback(() => {
-    if (NO_VISUALIZER_ROUTES.has(pathname)) return
+  const onShowVisualizer = useCallback(
+    e => {
+      e.stopPropagation()
```

Comment:
I'd call stopPropagation after showVisualizer just to make sure that happens as uninterrupted as possible